src/Bicep.Core/Diagnostics/DiagnosticBuilder.cs

```diff
@@ -1970,6 +1970,46 @@ public ErrorDiagnostic RuntimeValueNotAllowedInFunctionDeclaration(string? acces
                 TextSpan,
                 "BCP350",
                 $"Value of type \"{valueType}\" cannot be assigned to an assert. Asserts can take values of type 'bool' only.");
+
+            public ErrorDiagnostic ExpectedSymbolListOrWildcard() => new(
+                TextSpan,
+                "BCP351",
+                "Expected left brace ('{') or asterisk ('*') character at this location.");
+
+            public ErrorDiagnostic ExpectedExportedSymbolName() => new(
+                TextSpan,
+                "BCP352",
+                "Expected the name of an exported symbol at this location.");
+
+            public ErrorDiagnostic ExpectedSymbolAliasIdentifier() => new(
+                TextSpan,
+                "BCP353",
+                "Expected a valid symbol alias at this location.");
```

Comment:
>symbol alias [](http://example.com/codeflow?start=34&length=12)

What's a "symbol alias"? Should we call it a namespace identifier or something similar?

Reply:
I replaced this message with "Expected a valid namespace identifier..." (for wildcard imports) and "Expected a valid type identifier..." (for cherry-pick imports) to match the pattern we use for other symbols with invalid identifiers.

Reply:
Sounds good!